<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameUIClass</class>
 <widget class="QMainWindow" name="GameUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item alignment="">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="menuPage">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" alignment="|Qt::AlignmentFlag::AlignVCenter">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>100</horstretch>
            <verstretch>100</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Simon Says</string>
          </property>
          <property name="pixmap">
           <pixmap>../../../../../../../ProjectModernCpp/ClientQT/ClientQT/Logo.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0" alignment="|Qt::AlignmentFlag::AlignVCenter">
         <widget class="QComboBox" name="comboBox">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>40</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::LeftToRight</enum>
          </property>
          <item>
           <property name="text">
            <string>Easy</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Medium</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hard</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="startButton">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Start Game</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gamePage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
             <widget class="QLCDNumber" name="lcdMaxScore">
              <property name="frameShape">
               <enum>QFrame::Shape::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Sunken</enum>
              </property>
              <property name="digitCount">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="highScoreLabel">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LayoutDirection::RightToLeft</enum>
              </property>
              <property name="text">
               <string>HIGH SCORE</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLCDNumber" name="lcdLevel">
              <property name="layoutDirection">
               <enum>Qt::LayoutDirection::RightToLeft</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Plain</enum>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="smallDecimalPoint">
               <bool>true</bool>
              </property>
              <property name="digitCount">
               <number>1</number>
              </property>
              <property name="mode">
               <enum>QLCDNumber::Mode::Hex</enum>
              </property>
              <property name="segmentStyle">
               <enum>QLCDNumber::SegmentStyle::Outline</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="scoreLabel">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>SCORE</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <property name="rightMargin">
             <number>9</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="1" column="2">
             <widget class="QPushButton" name="greenButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color:rgb(85, 255, 0);
    border-radius: 5px;
    padding: 5px;
}

QPushButton:pressed {
    background-color: rgb(48, 144, 0)
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="blueButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color:rgb(0, 170, 255);
    border-radius: 5px;
    padding: 5px;
}

QPushButton:pressed {
    background-color: rgb(0, 112, 168)
} </string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="redButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color:rgb(170, 0, 0);
    border-radius: 5px;
    padding: 5px;
}

QPushButton:pressed {
    background-color: rgb(126, 0, 0)
} 
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="yellowButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color:rgb(255, 255, 0);
    border-radius: 5px;
    padding: 5px;
}

QPushButton:pressed {
    background-color: rgb(184, 184, 0)
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GameUI.qrc"/>
 </resources>
 <connections/>
</ui>
